x-pack/legacy/plugins/alerting/server/alert_instance/alert_instance.test.ts

```diff
@@ -192,7 +192,7 @@ describe('updateLastScheduledActions()', () => {
       state: {},
       meta: {
         lastScheduledActions: {
-          date: new Date(),
+          date: new Date().toISOString(),
```

Comment:
It worked because date was being being evaluated as a string before hand, causing us to have mixed types.
Adding the typing revealed the issue which is now addressed.